index.html

```diff
@@ -5,24 +5,46 @@
 	<title>netflix</title>
 	<link href="styles/reset.css" type="text/css" rel="stylesheet">
 	<link href="styles/main-layout.css" type="text/css" rel="stylesheet">
+	<link href="styles/base.css" type="text/css" rel="stylesheet">
+	<link href="styles/font.css" type="text/css" rel="stylesheet">
+	<link href="styles/logo.css" type="text/css" rel="stylesheet">
+	<link href="styles/btn.css" type="text/css" rel="stylesheet">
+	<link href="styles/utility.css" type="text/css" rel="stylesheet">
+	<link href="styles/input-search.css" type="text/css" rel="stylesheet">
 </head>
 <body>
 <div class="c-main-layout">
-	<header class="c-main-layout__header" style="background: lightblue">
+	<header class="c-main-layout__header">
 		<div class="c-main-layout__wrapper">
-			hhklkk;l;
+			<div class="c-logo"></div>
+			<div class="c-main-layout__header-add-btn">
+				<button class="c-btn">+ <span class="u-only-desktop">add movie</span></button>
+			</div>
+			<div class="c-main-layout__header-search-bar">
+				<h1>find your movie</h1>
+				<form>
+					<ul class="c-main-layout__header-form">
+						<li>
+							<input class="c-input-search" type="text" name="movie" placeholder="What do you want to watch?">
```

Comment:
1) This tag should be closed "/>"
`<input ... />`
2) Why do you need the attribute "name"?